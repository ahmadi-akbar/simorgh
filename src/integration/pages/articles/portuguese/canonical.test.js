/**
 * @service portuguese
 * @pathname /portuguese/articles/c72mg3j3x7eo
 */

import { runFlourishCanonicalEmbedTests } from '../../../common';
import runInlineLinkTests from '../inlineLink';

describe('Canonical', () => {
  describe(pageType, () => {
    runFlourishCanonicalEmbedTests();
    runInlineLinkTests();
    // WE NEED A LTR INLINE LINK TEST BUT NEED TO MAKE FIXTURE DATA FOR THIS
  });
});
