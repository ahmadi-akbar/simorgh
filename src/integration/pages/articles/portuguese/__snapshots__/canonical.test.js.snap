// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Articles Flourish Embed Story should match snapshot 1`] = `null`;
